### ISSUE TITLE
Ash column height calculation

### ISSUE THREAD

**Issue by ivanbusthomi** [inasafe/inasafe-realtime#151]
### Problem
At the moment, there is no information in the upload page about the reference of the ash column height. The height of the ash column given usually calculated from the vent of the volcano, while the information shown in the report is calculated from above sea level. This might confuse user who upload the data and wrong information to be put while uploading.
for example user who upload ash with the column height of 3000m above the vent will give information like this
![image](https://user-images.githubusercontent.com/7351977/34253332-afa7fc4e-e679-11e7-97fb-b5108850c013.png)

and the report that they are getting is like this
![selection_004](https://user-images.githubusercontent.com/7351977/34253261-6e59d280-e679-11e7-8389-669fdec04b22.png)

### Expected Behaviour
There should be clear and correct information that is shown and in the report

### Proposed Solution
* Put additional information in the form stating that the height is calculated from the vent
* Add the column height from the vent to the vent height from sea level, so that the value is properly above sea level as shown in the report.

CC @Charlotte-Morgan @lucernae @gubuntu

**Comment by Charlotte-Morgan** [comment]
Hi @ivanbusthomi - thank you for this ticket.
If I understand you are proposing:
The user enters the eruption height on the form (height above the vent)
The eruption column height is then calculated from the value entered + the vent height. This value is shown as asl on the report.
eg: for Agung, eruption height + vent height = column height
3000 + 4142 = 7142

**Comment by lucernae** [comment]
We have the vent height data from volcano data that mas @samnawi gave me and it is on the database. so we can calculate that.
Is this ticket ready for me to implement @ivanbusthomi ?

**Comment by Charlotte-Morgan** [comment]
@ivanbusthomi  - please confirm

**Comment by ivanbusthomi** [comment]
@lucernae , I am sorry for the late response. 
> We have the vent height data from volcano data that mas @samnawi gave me and it is on the database. so we can calculate that.

Yes, I think that is the proper method.

**Comment by lucernae** [comment]
@ivanbusthomi @Charlotte-Morgan I merged the changes. I had a couple of worries though.
The volcanoes data provided in InaSAFE Realtime uses field called `elevation` for vent height. Is this become an issue? I Imagine volcano elevation and vent height could mean different things?

Also, I proposed that we add more read only field in the form to avoid misunderstanding, something like this:

<img width="998" alt="screen shot 2018-01-23 at 04 27 17" src="https://user-images.githubusercontent.com/831865/35246676-1870d374-fffa-11e7-8fde-5a4d35166d6f.png">

Basically we shows `volcano height` as readonly field (because it was taken from database).
Then we shows `calculated eruption height (asl)` too. So the user understand that the eruption height is coming from the addition of volcano height and eruption height from vent.

I also propose we do something similar in the ash list in landing page, to show these 3 data.

**Comment by lucernae** [comment]
Also note that the changes in how we calculate eruption height will make previous report shows inconsistent number because eruption height now represent a different thing (from vent).

**Comment by gubuntu** [comment]
can we close?

**Comment by Charlotte-Morgan** [comment]
yes - that is why we have version control :)
